tests/checkBoxes.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { test, expect } from '@playwright/test';
+import exp from 'constants';
+
+test.beforeEach(async({page}) => {
+    await page.goto('https://petclinic.bondaracademy.com/welcome')
+})
+
+test.describe('Checkboxes', () => {
+    test.beforeEach(async ({page}) => {
+        await page.getByRole('button', {name: " Veterinarians"}).click()
+        await page.getByRole('link', {name: " All"}).click() //1. Select the VETERINARIANS menu item in the nav bar, then select "All"
+    })
+
+    test('validate selected specialties', async ({page}) => {
+        const vetDoctorTableHeader = page.locator('app-vet-list h2')
+        await expect(vetDoctorTableHeader).toHaveText('Veterinarians') //2. Assert Veterinarians text is displayed above table
+        const vetTableRowDr = page.getByRole('row', {name: " Helen Leary "})
+        await vetTableRowDr.getByRole('button', {name: 'Edit Vet'}).click() //3. Select Helen Leary and click Edit Vet
```

Comment:
lines 17 and 18 could be a single line since `vetTableRowDr` was not used anywhere else in the code

Reply:
Changed